util/secscan/v4/api.py

```diff
@@ -204,5 +204,6 @@ def is_valid_response(action, resp={}):
         try:
             validate(resp, schema, resolver=resolver)
             return True
-        except Exception:
+        except Exception as e:
+            logger.exception(e)
```

Comment:
`logger.exception` will already add the exception contents; please put a descriptive message on this

Reply:
Fixed